docs/sources/alerting/alerting-rules/create-mimir-loki-managed-recording-rule.md

```diff
@@ -29,22 +29,29 @@ refs:
       destination: /docs/grafana-cloud/alerting-and-irm/alerting/fundamentals/alert-rules/annotation-label/
 ---
 
-# Create recording rules
+# Configure recording rules
+
+{{< admonition type="note" >}}
+In Grafana Cloud, you can only create data source-managed recording rules.
+
+In Grafana OSS, you can create both Grafana-managed and data source-managed recording rules if you enable the `grafanaManagedRecordingRules` feature flag.
```

Comment:
@soniaAguilarPeiron ,  Out of curiosity, is there any plan to enable the `grafanaManagedRecordingRules` feature flag in Grafana Cloud?

Reply:
there is but not yet